diff -urN libpod-1.8.1_orig/vendor/github.com/openshift/imagebuilder/dispatchers.go libpod-1.8.1/vendor/github.com/openshift/imagebuilder/dispatchers.go
--- libpod-1.8.1_orig/vendor/github.com/openshift/imagebuilder/dispatchers.go	2020-03-11 15:16:13.000000000 +0100
+++ libpod-1.8.1/vendor/github.com/openshift/imagebuilder/dispatchers.go	2020-03-15 00:31:04.307075646 +0100
@@ -132,7 +132,11 @@
 	last := len(args) - 1
 	dest := makeAbsolute(args[last], b.RunConfig.WorkingDir)
 	if len(flagArgs) > 0 {
-		for _, arg := range flagArgs {
+        for _, a := range flagArgs {
+            arg, err := ProcessWord(a, b.Env)
+            if err != nil {
+                return err
+            }
 			switch {
 			case strings.HasPrefix(arg, "--chown="):
 				chown = strings.TrimPrefix(arg, "--chown=")
@@ -158,7 +162,12 @@
 	var chown string
 	var from string
 	if len(flagArgs) > 0 {
-		for _, arg := range flagArgs {
+        for _, a := range flagArgs {
+            arg, err := ProcessWord(a, b.Env)
+            if err != nil {
+                return err
+            }
+
 			switch {
 			case strings.HasPrefix(arg, "--chown="):
 				chown = strings.TrimPrefix(arg, "--chown=")
